ts/components/ui/cards/payment/PaymentCardsCarousel.tsx

```diff
@@ -11,7 +11,7 @@ import {
 } from "./PaymentCardSmall";
 
 export type PaymentCardsCarouselProps = WithTestID<{
-  cards: Array<PaymentCardSmallProps>;
+  cards: Array<PaymentCardSmallProps | undefined>;
```

Comment:
```suggestion
  cards: Array<PaymentCardSmallProps>;
```
I think it is better to filter out null elements from the source instead of accepting them in the props. What do you think?

Reply:
well sure, why not? the original idea war to avoid having one more loop in render but after all it's a minor improvement at the cost of possible stability so sure!